contracts/protocol/bases/mixins/Custody.sol

```diff
@@ -125,7 +126,7 @@ abstract contract Custody is FundsManager {
         offerVault.amount += amountToTransferToOfferVault;
 
         if (_externalCall && returnedAmount > 0) {
-            transferERC20FromProtocol(exchangeToken, payable(msg.sender), returnedAmount); // not using msgSender() since caller is FermionFNFT contract
+            transferERC20FromProtocol(exchangeToken, payable(ContextLib._msgSender()), returnedAmount);
```

Comment:
This was ok as it was, since we check in line 83 that the caller is the FNFT, so this will never be invoked via metaTransactions and using `_msgSender()` is redundant.